odl/tomo/backends/astra_cuda.py

```diff
@@ -123,6 +122,11 @@ def astra_cuda_forward_projector_call(vol_data, geometry, proj_space,
         else:
             out.assign(tmp)
```

Comment:
This is likely slower than simply not calling `proj_space.element(` above and instead using the `__setitem__` operator here:

```
np.rollaxis(astra.data3d.get(sino_id), 0, 3)
out[:] = tmp
```